# Question

Will the code work? Why or why not?

```js
const getUsers = async () => {
  const url = 'https://jsonplaceholder.typicode.com/users';
  const users = await fetch(url).then(r => r.json()).catch((err) => []);
  console.log(users)
  return users;
}
getUsers()
```

Again, running the code gives you the answer, so try to figure it out on your own first!

# Answer
The code will work as `await` will ensure that both the fetch call and the JSON parsing complete before proceeding. It handles any errors by returning an empty array if an error occurs. Therefore, the `users` variable will contain the fetched data or an empty array if the fetch fails.

# Teacher Feedback

Yup!

Score: 3/3